app/sentry.ts

```diff
@@ -20,4 +20,15 @@ Sentry.init({
   environment,
   tracesSampleRate,
   profilesSampleRate,
+  beforeBreadcrumb: breadcrumb =>
```

Comment:
This is just to stop duplication console logs in the application. It was annoying me.